knimin/lib/data_access.py

```diff
@@ -4071,7 +4072,32 @@ def get_targeted_plate_list(self):
             TRN.add(sql)
             return [dict(row) for row in TRN.execute_fetchindex()]
 
-    def pool_plates(self, pools, name, volume):
+    def get_quantified_targeted_plate_list(self):
+        """Gets the list of all quantified targeted plates
+
+        Returns
+        -------
+        list of dict
+            {id : int, name : str, date : datetime, num_samples: int}
+            Plate id, plate name, date and number of samples
+        """
+        with TRN:
+            # Magic number 0.0001: since we are dealing with floats, we need
+            # the ones that have different from 0. All those values are always
+            # positive, and the minimum threshold that the user can choose is
+            # 0.001. By using 0.0001 we ensure that we count correctly
```

Comment:
Not sure how to get it in SQL, and I don't think we need to be that fine grained.